config/api.yml

```diff
@@ -176,8 +176,8 @@
       - :name: delete
         :identifier: alert_definition_delete
   :alerts:
-    :description: Alerts
-    :identifier: alert
+    :description: Alert Statuses
+    :identifier: alert_status
```

Comment:
Seems like this should just be changed to `miq_alert`, based on what the previous key was:

https://github.com/ManageIQ/manageiq/pull/20438/files#diff-cdcd147fb85ef58d008a4ad0cfa3ddfea649382cb062407e2342aff07362b6cfL2054

Also what I did in my PR:

https://github.com/ManageIQ/manageiq-api/pull/965/files#diff-0f4eeb68b6115c7dcee6074cbf57d5e60b4f0abd7abe5b2ec5b1f8cab58d8d35L180

Reply:
If you look at the stuff below, it says `alert_status_*` for all child features + the `klass` is also `MiqAlertStatus` so here the top route and its naming was wrong (IMO).

Reply:
However, unlike the name change above, I could see this having issues when a user upgrades because what was previously associated with one `MiqProductFeature`(?) is now associated with another.

Does this change also require a migration?

Reply:
Well, I can go with `miq_alert` which would not break compatibility but that's not the correct feature for this class name. Also there's no [migration](https://github.com/ManageIQ/manageiq-schema/pull/509) for [this](https://github.com/ManageIQ/manageiq/pull/20438/files#diff-cdcd147fb85ef58d008a4ad0cfa3ddfea649382cb062407e2342aff07362b6cfR2002) rename. cc @h-kataria

Reply:
@NickLaMuro https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/miq_product_features.yml#L2025-L2139 features do not have corresponding screens in UI these features were added for API usage only, and remain unchanged with my Control explorer work.
See screenshots from before the Control explorer split
<img width="417" alt="screen_shot_2020-11-20_at_11 40 11_am" src="https://user-images.githubusercontent.com/3450808/99828617-2f2c0680-2b29-11eb-829f-6bdac55139b9.png">

<img width="300" alt="screen_shot_2020-11-20_at_11 39 27_am" src="https://user-images.githubusercontent.com/3450808/99828630-318e6080-2b29-11eb-84c1-3cc410f51e85.png">

All you need to do is fix the identifier/description for `alert`